src/Generator/CodeGenerator.php

```diff
@@ -471,7 +471,7 @@ private static function addImportForType($type, $namespace, array &$imports): vo
      */
     private static function isAliased($name, array $imports): bool
     {
-        $aliases = array_keys($imports);
+        $aliases = array_filter(array_keys($imports), 'is_string');
```

Comment:
I feel that the commit message does not really reflect the actual code change... What else than keys can the imports be?

Reply:
The imports are the values of the array. The array_key is either the alias, or a number in case there is none.

Reply:
```
use NameSpaced\Import\Collection;
use NameSpaced\Import\Alias as ABC;
```
would be
```
[
    0 => 'NameSpaced\Import\Collection',
   'ABC' => 'NameSpaced\Import\Alias'
]
```